account_vat_on_payment/account_bank_statement.py

```diff
@@ -0,0 +1,299 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2011-2012 Domsense s.r.l. (<http://www.domsense.com>).
+#    Copyright (C) 2014 Agile Business Group sagl (<http://www.agilebg.com>)
```

Comment:
You may want to write your name here. Also, please use short headers, like https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py